x-pack/plugins/observability_solution/observability_ai_assistant_app/server/functions/query/esql_docs/esql-functions-overview.txt

```diff
@@ -0,0 +1,118 @@
+Functions overview
+
+
+Aggregate functions
+AVG
+COUNT
+COUNT_DISTINCT
+MAX
+MEDIAN
+MEDIAN_ABSOLUTE_DEVIATION
+MIN
+PERCENTILE
+[preview]
+This functionality is in technical preview and may be changed or removed in a future release. Elastic will work to fix any issues, but features in technical preview are not subject to the support SLA of official GA features.
+ST_CENTROID_AGG
```

Comment:
I wonder if it's clear to the LLM which part of the text blob this line refers to?

Reply:
There might be room for improvement here, but I'm not sure if we can automate it, so probably not worth the effort. It will see it in the function specific file regardless.